.gitignore

```diff
@@ -117,4 +117,7 @@ dmypy.json
 dump.rdb
```

Comment:
I like to use https://www.conventionalcommits.org/en/v1.0.0/

It's nice and make reding the commit list easier,

Also you can use keyword to close issues

https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords

And I like to add the keyword in a second line as a description.

You message would be 
```
feat: implement a status middleware

fix #106
```